multigeiger/multigeiger.ino

```diff
@@ -943,83 +943,81 @@ void sendData2http(const char* host, int sendwhat, unsigned int hvpulses, unsign
   unsigned char ttnData[20];
   int cnt;
-  if(sendwhat == SEND_CPM) {
+  if (sendwhat == SEND_CPM) {
     // first the number of counts
-    ttnData[0] = (GMC_counts_2send >>24) & 0xFF;
-    ttnData[1] = (GMC_counts_2send >>16) & 0xFF;
-    ttnData[2] = (GMC_counts_2send >>8) & 0xFF;
+    ttnData[0] = (GMC_counts_2send >> 24) & 0xFF;
+    ttnData[1] = (GMC_counts_2send >> 16) & 0xFF;
+    ttnData[2] = (GMC_counts_2send >> 8) & 0xFF;
     ttnData[3] = GMC_counts_2send & 0xFF;
     // now 3 bytes for the time in ms for this numer of counts (max ca. 4 hours)
     ttnData[4] = (timediff >> 16) & 0xFF;
     ttnData[5] = (timediff >> 8) & 0xFF;
-    ttnData[6] = timediff & 0xFF;                         
+    ttnData[6] = timediff & 0xFF;
     // next two bytes are software version
-    ttnData[7] = (lora_software_version>>8)&0xFF;
-    ttnData[8] = lora_software_version&0xFF;
+    ttnData[7] = (lora_software_version >> 8) & 0xFF;
+    ttnData[8] = lora_software_version & 0xFF;
     // next byte is the tube version
     ttnData[9] = tubes[TUBE_TYPE].nbr;
     cnt = 10;
-    lorawan_send(1,ttnData,cnt,false,NULL,NULL,NULL);
+    lorawan_send(1, ttnData, cnt, false, NULL, NULL, NULL);
   };
-  if(sendwhat == SEND_BME) {
-    ttnData[0] = ((int)(bme_temperature*10)) >> 8;
-    ttnData[1] = ((int)(bme_temperature*10)) & 0xFF;
-    ttnData[2] = (int)(bme_humidity*2);
-    ttnData[3] = ((int)(bme_pressure/10)) >> 8;
-    ttnData[4] = ((int)(bme_pressure/10)) & 0xFF;
+  if (sendwhat == SEND_BME) {
+    ttnData[0] = ((int)(bme_temperature * 10)) >> 8;
+    ttnData[1] = ((int)(bme_temperature * 10)) & 0xFF;
+    ttnData[2] = (int)(bme_humidity * 2);
+    ttnData[3] = ((int)(bme_pressure / 10)) >> 8;
+    ttnData[4] = ((int)(bme_pressure / 10)) & 0xFF;
     cnt = 5;
-    lorawan_send(2,ttnData,cnt,false,NULL,NULL,NULL);
+    lorawan_send(2, ttnData, cnt, false, NULL, NULL, NULL);
   }
 }
 #endif
 
 /**
  * Handle web requests to "/" path.
  */
-void handleRoot(void)
-{
+void handleRoot(void) {
   // -- Let IotWebConf test and handle captive portal requests.
-  if (iotWebConf.handleCaptivePortal())
-  {
+  if (iotWebConf.handleCaptivePortal()) {
     // -- Captive portal requests were already served.
     return;
   }
-  const char* index =
-"<!DOCTYPE html>"
-"<html lang='en'>"
-"<head>"
-  "<meta name='viewport' content='width=device-width, initial-scale=1, user-scalable=no' />"
-  "<title>MultiGeiger Configuration</title>"
-"</head>"
-"<body>"
-  "<h1>Configuration</h1>"
-  "<p>"
+  const char *index =
+    "<!DOCTYPE html>"
+    "<html lang='en'>"
+    "<head>"
+    "<meta name='viewport' content='width=device-width, initial-scale=1, user-scalable=no' />"
+    "<title>MultiGeiger Configuration</title>"
+    "</head>"
+    "<body>"
+    "<h1>Configuration</h1>"
+    "<p>"
     "Go to the <a href='config'>configure page</a> to change settings or update firmware."
-  "</p>"
-"</body>"
-"</html>\n";
+    "</p>"
+    "</body>"
+    "</html>\n";
```

Comment:
automatted formatting kills our string-storage space-saving indentation here. but guess we can live with that.

Reply:
That's OK